scripts/linting/_check-etags

```diff
@@ -11,12 +11,9 @@ check_presence () {
 
 }
 
-check_presence analysis.js
```

Comment:
This should be replaced by a blazor file?

Reply:
Good catch; done.

Reply:
`check_presence BlazorWorker.js` already exists within `_check-etags`

(fwiw, the removal of `check_presence analysis.js` is here _twice_ because it was in there twice originally for some reason: https://github.com/darklang/dark/blob/c4347492a30a8f01c6e82dc89e4f1a0c7170aac8/scripts/linting/_check-etags#L14-L18)